Formula/r/rpmspectool.rb

```diff
@@ -15,7 +15,8 @@ class Rpmspectool < Formula
   depends_on :linux
   depends_on "python-argcomplete"
   depends_on "python-pycurl"
-  depends_on "python@3.11"
+  depends_on "python-setuptools"
```

Comment:
Remind me why this is still needed given that `python@3.12` still has a `setuptools` resource?

Reply:
it isn't actually shipped with `python@3.12` any longer, we still have the resource but only to build `pip` from source in a temporary venv (same with `flit-core`)

https://github.com/Homebrew/homebrew-core/blob/7de456363e6e1d773cc065219fff8f5c24d4348c/Formula/p/python%403.12.rb#L279-L285

Reply:
In that case, maybe this is no longer necessary?

https://github.com/Homebrew/homebrew-core/blob/7de456363e6e1d773cc065219fff8f5c24d4348c/Formula/p/python%403.12.rb#L24-L26

CC @Bo98

Reply:
yea shouldn't be

Reply:
Pretty sure it's a `sysconfigdata` thing and not setuptools specifically.

With that said, we should do better here as CLT requirement is only really necessary for things that interface with C compiling, which is not that much.

Possible idea is a relocation-like thing where we replace the SDK reference with a placeholder and write in the new SDK in `post_install`.